cmd/soroban-cli/src/commands/events.rs

```diff
@@ -156,26 +156,16 @@ impl Cmd {
 
         let response = self.run_against_rpc_server(None, None).await?;
 
-        for event in &response.events {
-            match self.output {
-                // Should we pretty-print the JSON like we're doing here or just
-                // dump an event in raw JSON on each line? The latter is easier
-                // to consume programmatically.
-                OutputFormat::Json => {
-                    println!(
-                        "{}",
-                        serde_json::to_string_pretty(&event).map_err(|e| {
-                            Error::InvalidJson {
-                                debug: format!("{event:#?}"),
-                                error: e,
-                            }
-                        })?,
-                    );
-                }
-                OutputFormat::Plain => println!("{event}"),
-                OutputFormat::Pretty => event.pretty_print()?,
```

Comment:
The hope of the issue was to address that "Events in pretty print mode print as Rust debug format," and same in the plain mode too I think. The intent is to fix that, rather than return only JSON. So each topic and the value should print as JSON, but the rest of the format should stay the same.

See:
- https://github.com/stellar/stellar-cli/issues/1615

Reply:
ok sure, thank you for letting me know, i'll fix that as soon as possible